package.json

```diff
@@ -48,8 +48,14 @@
     "@dreipol/scss-utils": "^0.1.0"
   },
   "devDependencies": {
-    "@dreipol/stylelint-config": "^4.0.0",
     "node-sass": "4.9.3",
-    "stylelint": "^9.6.0"
+    "stylelint": "^9.6.0",
```

Comment:
Please add them in the order in which they are after installation. Else we will have changes after each local installation